online-shop/src/components/NavBar.tsx

```diff
@@ -0,0 +1,48 @@
+import React, {Component} from 'react';
+import {Link} from 'react-router-dom';
+import iconPage from '../icon.png';
+import '../productModeling/navBar.css'
+import styled from 'styled-components';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { library, icon } from '@fortawesome/fontawesome-svg-core'
+import { faCamera,faShoppingCart } from '@fortawesome/free-solid-svg-icons'
+
+library.add(faCamera,faShoppingCart)
```

Comment:
To not lose track of what you added and where, I would suggest having a `icons.ts` file next to your `index.ts`, do all the "add" calls there and then import that file in the `index.ts`.